app/experimenter/nimbus-ui/package.json

```diff
@@ -14,7 +14,7 @@
     "storybook": "start-storybook -p 3001",
     "build-storybook": "build-storybook",
     "eject": "react-scripts eject",
-    "types": "NODE_TLS_REJECT_UNAUTHORIZED=0 apollo codegen:generate --target typescript --endpoint https://localhost/api/v5/graphql types"
+    "types": "NODE_TLS_REJECT_UNAUTHORIZED=0 apollo codegen:generate --target typescript --endpoint https://localhost/api/v5/graphql --outputFlat src/types"
```

Comment:
Wrestled with the type generation a bit and had to make this change. Some types were generated outside of the source tree, which broke Storybook at least.

Reply:
Ooh good catch for Storybook, hadn't tested that.